lib/contentful-management.ts

```diff
@@ -17,10 +27,34 @@ import { createCMAHttpClient, ClientParams } from './create-cma-http-client'
  * })
  * ```
  */
-export function createClient(params: ClientParams): ClientAPI {
-  const http = createCMAHttpClient(params)
-  const api = createContentfulApi({
-    http: http,
-  })
-  return api
+
+function createClient(params: ClientParams): ClientAPI
+function createClient(
+  params: ClientParams,
+  opts: {
+    plainClient?: true
```

Comment:
It will only work if `planclient: true` is given, so I'd remove the question mark.

Reply:
Change to `type: 'plain'`